src/PowerShellEditorServices.Protocol/MessageProtocol/MessageWriter.cs

```diff
@@ -58,20 +58,34 @@ public async Task WriteMessage(Message messageToWrite)
                 this.messageSerializer.SerializeMessage(
                     messageToWrite);
 
-            this.logger.Write(
-                LogLevel.Verbose,
-                $"Writing {messageToWrite.MessageType} '{messageToWrite.Method}'" +
-                (!string.IsNullOrEmpty(messageToWrite.Id) ? $" with id {messageToWrite.Id}" : string.Empty));
-
-            // Log the JSON representation of the message
-            this.logger.Write(
-                LogLevel.Diagnostic,
-                string.Format(
-                    "WRITE MESSAGE:\r\n\r\n{0}",
+            // Log message info - initial capacity for StringBuilder varies depending on whether
+            // the log level is Diagnostic where JsonRpc message payloads are logged and 
+            // vary from 1K up 225K chars.  When not logging message payloads the typical response
```

Comment:
```suggestion
            // vary from 1K up to 225K chars.  When not logging message payloads the typical response
```
Is this what you meant?

Reply:
Yup, will fix tonight.  BTW that is the largest response I've observed so far.  :-)